source/_integrations/eltako.markdown

```diff
@@ -7,17 +7,23 @@ ha_category:
   - Switch
 ha_domain: eltako
 ha_integration_type: brand
+works_with:
+  - matter
 ha_platforms:
   - switch
 ha_iot_standard: matter
 ha_brand: true
 ---
 
-[Eltako](https://www.eltako.com) is committed to making sure their products are up-to-date and ready to use in Home Assistant.
 
-Eltako Matter devices work locally and integrate seamlessly with the Matter integration in Home Assistant. As all connectivity is happening locally, status updates and controlling your devices happen instantly in Home Assistant.
+{% include integrations/wwha.md url="https://www.eltako.com" name="Eltako" %}
 
-{% my add_matter_device badge domain=page.ha_domain %}
+## Supported devices
 
-[Learn more about Matter in Home Assistant.](/integrations/matter/)
+The following devices are supported:
 
+- [ELTAKO Universal Dimming Actuator EUD65NPN-IPM](https://www.eltako.com/en/catalog/products/1761/eud64npn-ipm)
```

Comment:
is there a reason the device types use heading-style capitalization? From what I can see, their own user documentation uses sentence-style capitalization. https://www.eltako.com/fileadmin/downloads/en/_bedienung/ESR64NP-IPM%2030064015-1_gb.pdf

Reply:
Removed ELTAKO since it is redundant being on Eltako's integration page.